Lib/webbrowser.py

```diff
@@ -113,7 +113,7 @@ def _synthesize(browser, update_tryorder=1):
         controller = copy.copy(controller)
         controller.name = browser
         controller.basename = os.path.basename(browser)
-        register(browser, None, controller, update_tryorder)
+        register(browser, None, instance=controller, preferred=update_tryorder)
```

Comment:
`update_tryorder` is 1 or -1. `preferred` is a boolean parameter. 1 and -1 both are true in boolean context.

I think that the `update_tryorder` parameter of `_synthesize()` should be replaced with boolean `preferred` as in `register()` (see (bpo-24241)[https://bugs.python.org/issue24241]).

Reply:
Update tryorder is now a boolean as requested.